woocommerce/google-listings-and-ads
# Add Woo gTag remarketing and conversion signals

### Changes proposed in this Pull Request:
Add gTag info in home/product/cart/purchase/add_to_cart page

Closes # .

Add remarketing and conversion signals in the gTag in the following pages (home, product, add to cart action, cart, purchase) 
More details: https://docs.google.com/document/d/1UouhR5Hj-9gc5jPhxWPLX8-nXhoHG5jMeiva7iQrhy8/edit#


### Screenshots:



![Screen Shot 2022-05-23 at 1 23 53 PM](https://user-images.githubusercontent.com/97628051/169900600-766e364b-8cb6-4636-9258-aa421e2668eb.png)

<img width="689" alt="Screen Shot 2022-05-24 at 1 10 15 PM" src="https://user-images.githubusercontent.com/97628051/170125548-f4d6f3ac-813c-4701-9dde-f0948bace02d.png">


![Screen Shot 2022-05-25 at 11 30 35 AM](https://user-images.githubusercontent.com/97628051/170341451-74f3617e-7383-429b-88e4-d1c659934322.png)

<img width="510" alt="Screen Shot 2022-05-24 at 1 09 25 PM" src="https://user-images.githubusercontent.com/97628051/170125592-2b7db095-9e3e-43de-b26f-81e6f64da846.png">

![Screen Shot 2022-05-25 at 11 31 49 AM](https://user-images.githubusercontent.com/97628051/170341480-1947ef52-0a3c-4b9b-a0ec-812f434561dc.png)


### Detailed test instructions:



1.  Connect Merchant Center and Ads accounts.
2.  Visit the public-facing store.
3.  View the page source code and confirm:

- In the each page
The gTag is preceded by the new HTML comment:
gtag("event", "page_view", {...})

- In the product page
The gTag is preceded by the new HTML comment:
gtag("event", "view_item", {...})	

- In the cart page
The gTag is preceded by the new HTML comment:
gtag("event", "page_view", {...})

- In the purchase page
The gTag is preceded by the new HTML comment:
gtag("event", "purchase", {...})
